### ISSUE TITLE
AddCephPrimaryStorage fail

### ISSUE THREAD

**Issue by youyk** [zstackorg/zstack#89]
DEBUG:zstacklib.utils.shell:ceph -f json auth get-or-create client.zstack mon 'allow r' osd 'allow *' 2>/dev/null
DEBUG:zstacklib.utils.shell:ceph df -f json
WARNING:cephprimarystorage.cephagent:long() argument must be a string or a number, not 'NoneType'
Traceback (most recent call last):
  File "/var/lib/zstack/virtualenv/cephp/lib/python2.7/site-packages/cephprimarystorage/cephagent.py", line 54, in wrap
    return func(*args, **kwargs)
  File "/var/lib/zstack/virtualenv/cephp/lib/python2.7/site-packages/cephprimarystorage/cephagent.py", line 229, in init
    self._set_capacity_to_response(rsp)
  File "/var/lib/zstack/virtualenv/cephp/lib/python2.7/site-packages/cephprimarystorage/cephagent.py", line 106, in _set_capacity_to_response
    total = long(df.stats.total_bytes_)
TypeError: long() argument must be a string or a number, not 'NoneType'

In my ceph:

```
[root@ceph-01 ~]# ceph df -f json|grep total
{"stats":{"total_space":720338944,"total_used":1861428,"total_avail":718477516},"pools":[{"name":"data","id":0,"stats":{"kb_used":0,"bytes_used":0,"max_avail":344441457598,"objects":0}},{"name":"metadata","id":1,"stats":{"kb_used":0,"bytes_used":0,"max_avail":344441457598,"objects":0}},{"name":"rbd","id":2,"stats":{"kb_used":0,"bytes_used":0,"max_avail":344441457598,"objects":0}},{"name":"pri-v-r-86b1fc9c6c224c71ad0d3e1b6d60a0ca","id":4,"stats":{"kb_used":0,"bytes_used":0,"max_avail":344441457598,"objects":0}},{"name":"pri-v-d-86b1fc9c6c224c71ad0d3e1b6d60a0ca","id":5,"stats":{"kb_used":0,"bytes_used":0,"max_avail":344441457598,"objects":0}},{"name":"pri-c-86b1fc9c6c224c71ad0d3e1b6d60a0ca","id":6,"stats":{"kb_used":0,"bytes_used":0,"max_avail":344441457598,"objects":0}}]}
```
There isn't **total_bytes**

**Comment by youyk** [comment]
AddCephBackupStorage has same issue.

**Comment by zxwing** [comment]
fixed in 9a29ec02efdd0091895158944165ab1c0464aa39(utility)
